### ISSUE TITLE
Encrypted data isn't saved in AR

### ISSUE THREAD

**Issue by DSIW** [attr-encrypted/attr_encrypted#163]
I use Rails 3.2.21 and Ruby 2.1.5 with devise-two-factor 1.0.2 and attr_encrypted 1.3.4.

This User model is stored in separate database for every customer (tenant_1,...) and will be loaded in the ApplicationController via `establish_connection`. That works fine. But if I use your gem in production mode, I can't save the encrypted otp_secret of the user. Everything is ok in development mode. Every other attribute is successfully saved in production and development mode.

For demonstration in production mode:

``` ruby
class User
  attr_encrypted :otp_secret,
                 :key  => self.otp_secret_encryption_key,
                 :mode => :per_attribute_iv_and_salt
end

2.1.5 :033 > u = User.first
  User Load (0.7ms)  SELECT `users`.* FROM `users` LIMIT 1
 => #<User id: 1, encrypted_otp_secret: nil, encrypted_otp_secret_iv: nil, encrypted_otp_secret_salt: nil>
2.1.5 :034 > u.generate_otp_secret
 => "rwihy3uftnoesrns4dzs3ppczvfiapmq"
2.1.5 :035 > u.encrypted_otp_secret
 => "Vs0zHlNjarbgi2sVwGBJSG0DNYN9NpcFajTuft7NYbJy68OFZZnLQp3URRAa\nRN1D\n"
2.1.5 :036 > u.save!
   (0.5ms)  BEGIN
   (0.6ms)  UPDATE `users` SET `updated_at` = '2015-06-05 12:15:21' WHERE `users`.`id` = 1
   (1.2ms)  COMMIT
 => true
2.1.5 :037 > u
 => #<User id: 1, encrypted_otp_secret: nil, encrypted_otp_secret_iv: nil, encrypted_otp_secret_salt: nil>
```

Unfortunately the encrypted fields are nil. Do you know what the problem is or can you give me any advice? Thank you!

**Comment by saghaulor** [comment]
@DSIW did you ever figure out what the problem was?

**Comment by anicholson** [comment]
@saghaulor / @DSIW could be the same thing we hit: #217

**Comment by saghaulor** [comment]
I believe this problem is fixed by #294 

Please feel free to reopen this issue if it does not.
